### ISSUE TITLE
Error if no Message defined

### ISSUE THREAD

**Issue by martindsouza** [foex-open-source/fos-notifications#2]
I have a Title defined (ex: `Emp Updated`) but don't have / need a Message defined. If no Message is defined following JS error is raised:

```
Uncaught TypeError: can't access property "length", p is undefined
```

I think this error may be more to do with APEX messaging framework than this plugin. If this is the case not sure if a `&nbsp;` would work when the Message is blank.

**Comment by martindsouza** [comment]
Just checked and `&nbsp;` won't be parsed but a space ex: ` ` will work

**Comment by stefandobre** [comment]
Indeed, there are some inconsistencies there. Thanks for bringing it up.
Until we fix this, try the following: Leave the title empty, and put `Emp Updated` in the message. It will actually appear as the title.

**Comment by stefandobre** [comment]
This bug was fixed in FOS version 20.2.
You can download this version of the demo app and all plug-ins [here](https://fos.world/).
